pharus/interface.py

```diff
@@ -248,6 +248,10 @@ def _get_attributes(query) -> dict:
                         attribute_info.nullable,
                         attribute_info.default,
                         attribute_info.autoincrement,
+                        [
+                            dict({"text": str(x[0]), "value": x[0]})
+                            for x in (dj.U(attribute_name) & query).fetch()
+                        ],
```

Comment:
For backward compatibility and without having the overhead of having to perform this for every attribute on all routes, feels like we should introduce a new optional kwarg like `include_unique_values=False`. If specified, we can include in `attribute_headers` + in `attributes` like you have it. We probably need to limit this too somehow so that we don't run the risk of overflow. Maybe just a hard-coded limit for now? :shrug: 

This can also be simplified a bit to:

```suggestion
                        [
                            dict({"text": str(v), "value": v})
                            for (v,) in (dj.U(attribute_name) & query)
                        ],
```

Reply:
I have it optional now but I am not sure how to not include it if the size is to big